.github/workflows/test-macOS.yml

```diff
@@ -8,14 +8,15 @@ on:
   pull_request:
   schedule:
   - cron:  '0 8 * * *'
+  workflow_dispatch:
 
 permissions:
   contents: read
 
 jobs:
   run-tests:
     name: Python ${{ matrix.python-version }}
-    runs-on: macOS-10.15
+    runs-on: macOS-latest
```

Comment:
Since they removed 10.15 I figured testing on the latest wouldn't hurt.  Wasn't sure if there needs to be more than one version of mac for testing.